rest-backend/src/test/java/com/sap/psr/vulas/backend/rest/ApplicationControllerTest.java

```diff
@@ -421,8 +421,8 @@ public void readAllApplications() throws Exception {
     	// Read all apps for a non-existing token
     	mockMvc.perform(get("/apps")
      	    	  .header(Constants.HTTP_SPACE_HEADER, "does-not-exist"))
-                  .andExpect(status().isNotFound())
-                  .andExpect(content().contentType(contentTypeJson));
+                  .andExpect(status().isNotFound());
+                  //.andExpect(content().contentType(contentTypeJson));
```

Comment:
Why did that change in the context of upgrading to Spring Boot 2?

Reply:
I think there is a change in spring such that in 1.5.x a 404 response still had a body of type Json with content 'null', whereas there is no  body in such responses in 2.x